subprojects/studio-plugin/src/test/groovy/org/gradle/profiler/studio/plugin/StudioPluginIntegrationTest.groovy

```diff
@@ -38,14 +40,14 @@ class StudioPluginIntegrationTest extends StudioPluginTestCase {
         given:
         ServerConnection connection = server.waitForIncoming(Duration.ofSeconds(10))
         // Fail Gradle build by removing settings file
-        settingsFile.delete()
+        settingsFile << "garbage"
```

Comment:
This now does not match the comment above. Also, instead of appending `garbage` you could do:
```suggestion
        settingsFile.text = "garbage"
```

Reply:
thanks